src/ffi/mod.rs

```diff
@@ -973,7 +998,7 @@ pub extern "C" fn telio_get_commit_sha() -> *mut c_char {
 
 #[no_mangle]
 pub extern "C" fn telio_get_status_map(dev: &telio) -> *mut c_char {
-    trace!("acquiring dev lock");
+    telio_log_info!("telio_get_status_map entry with instance id: {}.", dev.id);
```

Comment:
how often it is going to be executed?

Reply:
Every time when `get_status_map()` is called. But this method is in fact used only on linux when executing `nordvpn mesh peer list` command.